L/LLVM/common.jl

```diff
@@ -499,6 +532,12 @@ function configure_build(ARGS, version; experimental_platforms=false, assert=fal
         Dependency("Zlib_jll"), # for LLD&LTO
         BuildDependency("LLVMCompilerRT_jll"; platforms=filter(p -> sanitize(p)=="memory", platforms)),
     ]
+    if v"15" <= VERSION
+        push!(dependencies,
+            ArchiveSource(
```

Comment:
I have been staring at these lines wondering how on Earth you managed to push an `ArchiveSource` into a vector of `AbstractDependency`-s.  Then I realised that was `VERSION` instead of `version` :smile:  I'm on it.